### PR TITLE
MWC 4K 2017 Recap

### PR DESCRIPTION
**Pull request by mangomizer**
Uploaded the MWC 4K 2017 Banner. To be used for future MWC 4K 2017 weekly recaps.

### CODE REVIEW

**Comment by TPGPL** [comment]
forgot to add a news template at the beginning I suppose?

File: news/2017-08-15-osu-mania-4k-world-cup-group-stage-recap.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "osu!mania 4K World Cup - Group Stage Recap"
+date: 2017-08-15 14:37:30 +0000
+---
+
+Hi everyone, mangomizer here! Summer may be ending, but the excitement continues! I’m here today of course, to catch up on the latest events regarding the hugely anticipated osu!mania 4K World Cup!
```
Comment (Ephemeralis) [inline news/2017-08-15-osu-mania-4k-world-cup-group-stage-recap.md]: drop "hi everyone mangomizer here" line from leading paragraph, rephrase to something more passive and focused on osu!mania 4K. maybe briefly reference (1 line) the super exciting stuff, these paragraphs are supposed to bait people into reading them, not explain who's doing the article and what its for

File: news/2017-08-15-osu-mania-4k-world-cup-group-stage-recap.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+![](/wiki/shared/news/banners/mania4k_logo_2017.jpg)
+
+This past weekend saw the conclusion of the Group Stage, where we had 184 talented players representing 32 countries squaring off against each other to qualify for the Round of 16. If you’ve missed the weekend matches, you can always rewatch the VODs [here](https://www.twitch.tv/osulive/videos/all), or settle for the weekly recap below! 
+
+This year’s Group Stage saw a grand total of 46 matches packed tightly into the span of 2 days. Courtesy of our beloved mappool selectors, we were treated to a truly diverse mappool featuring artists from all around the world. As expected, the mappool difficulty has increased substantially from last year, though this has not discouraged players in the slightest.
```
Comment (Ephemeralis) [inline news/2017-08-15-osu-mania-4k-world-cup-group-stage-recap.md]: "this" double dutches with previous paragraph, sub for 2017's group stage or something

File: news/2017-08-15-osu-mania-4k-world-cup-group-stage-recap.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+After such a fiercely exciting weekend, only 16 teams now remain. However, things will only get more challenging from here, as they will have to fight against each other over an increasingly difficult mappool.
+
+Props to the United Kingdom for being the only team to leave the Group Stage unscathed. How much longer will they be able to keep this up? I would also like to take this chance to congratulate New Zealand, for being the only unseeded team to make it through to the Round of 16. They put on a splendid performance this past weekend. As I understand, this is also the first time that ANY team from New Zealand has made it out of the Group Stage, so hopefully they will continue to impress, as I personally will be rooting for them! 
+
+This year’s installment of the MWC 4K promises to be an exciting one indeed. Every team is looking eager to take the trophy home, along with the glorious title of osu!mania Champion! Be sure to drop by the [osu!live Twitch Channel](https://www.twitch.tv/osulive) next Sunday to show your support for all the teams participating! 
+
+-mango
```
Comment (Ephemeralis) [inline news/2017-08-15-osu-mania-4k-world-cup-group-stage-recap.md]: u forgot the mizer

Comment (mangomizer) [inline news/2017-08-15-osu-mania-4k-world-cup-group-stage-recap.md]: no I didn't I'm MANGO! >.<

**Pull request closed without merging**